helm-chart/templates/nextcloud-aio-apache-persistentvolumeclaim.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   storageClassName: {{ .Values.STORAGE_CLASS }}
   {{- end }}
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
   resources:
     requests:
-      storage: {{ .Values.MAX_STORAGE_SIZE }}
+      storage: 1Gi
```

Comment:
So I googled about auto-resizing and it seems like there is no auto-resizing based on used storage. This is so dump because it means the admin needs to monitor this for each pvm as otherwise he/she risks that something runs out-of-space! Why is kubernetes does kubernetes not have this basic feature???

Reply:
Also now we need to make this adjustable at least foe the most-important vplumes that might exceed 1Gi...

Reply:
Or do you have an idea how to solve this problem systematically maybe?

Reply:
Hopefully fixed with https://github.com/nextcloud/all-in-one/pull/1848/commits/8562eb09d43734438c0da65a0a9162e0b11256aa

Reply:
There is auto resizing for some vendored cloud distributions (e.g. GKE). But yeah, for vanilla k8s the volume has to be manually recreated and migrated.

Reply:
Or something like https://longhorn.io/ can be used to easen volume handling.

Reply:
I see but I suppose this is up to the admin to choose?

Reply:
Exactly. Let admins handle their comes and just provide distinct variables.

Reply:
That is what I've done now :)